packages/blockchain-wallet-v4-frontend/src/scenes/app.tsx

```diff
@@ -105,8 +105,8 @@ const App = ({
                     <WalletLayout path='/airdrops' component={Airdrops} />
                     <WalletLayout path='/exchange' component={TheExchange} />
                     <WalletLayout path='/home' component={Home} />
-                    <WalletLayout path='/interest' component={Interest} exact />
-                    <WalletLayout path='/interest/history' component={InterestHistory} />
+                    <WalletLayout path='/rewards' component={Interest} exact />
+                    <WalletLayout path='/rewards/history' component={InterestHistory} />
```

Comment:
Are there any instances of this path outside of the wallet project? Such as in nginx or cloudflare? If so, I'll need to update those as well.

Reply:
nope, they are all internal to app